backend/drm/drm.c

```diff
@@ -50,6 +50,12 @@ bool check_drm_features(struct wlr_drm_backend *drm) {
 		return false;
 	}
 
+	if (drmGetCap(drm->fd, DRM_CAP_CRTC_IN_VBLANK_EVENT, &cap) ||
+			!cap) {
```

Comment:
the condition is small enough to be put on the last line correct?

Reply:
jip, fixed.